dist/create_bridge_config.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/env bash -x
+#!/bin/bash
```

Comment:
This appears to be a dupe of 01_bridgecf.config. Why is it necessary to have both?